### DISCUSSION TITLE
WindowTop Settings

### DISCUSSION THREAD

**Discussion by codeszhal** [WindowTop/WindowTop-App#248] [Q&A]
First I say thank you for this great software. You have increased my productivity. 
Actually I am used to using this program for less than two months. I have preconfigured the shotcut in the beginning and it helps me a lot. Now I want to view and edit the shortcut again in WindowTop settings window like the first time I did it. I have looked in the Windows Search button and it just show me the exe files to open the program. I have looked in the folder location when the app is installed and I still can't find it. Where I can find the WindowTop setting like I did it in the beginning?
![1](https://user-images.githubusercontent.com/17680514/147696937-fb742ddf-6c2d-4da4-a265-dca29a6465d4.gif)

**Comment by gileli121** [comment]
Hi @codeszhal 
Please look for the WindowTop icon in the tray menu.
You should find it here:
![image](https://user-images.githubusercontent.com/17680514/178907958-3628b33d-fa44-47e9-9da9-5411d196c670.png)

Or if is not there, click on the arrow and then you should find it:
![image](https://user-images.githubusercontent.com/17680514/178908057-e3f8d052-8cba-492b-aacd-ccc4fc6693b8.png)

After you found it, click on the icon.
This menu will open:
![image](https://user-images.githubusercontent.com/17680514/178908213-532fed60-e1db-4069-b1a3-e11d17d2f214.png)
In the menu, go to the "All settings" button and click on it.

It will open the quick start settings window that you showed.
The quick start settings is showing exactly what the wizard showed you. 
It uses the same pages that the wizard used.

Thanks for your confirmation that it helps you to increase your productivity!
May I ask what is the main feature you use that help your productivity?
